shared/xccdf/services/base.xml

```diff
@@ -411,4 +411,16 @@ operation, but unless used this service can be disabled.</rationale>
 <ref nist="CM-7" />
 </Rule>
 
+<Rule id="service_all_disabled" prodtype="rhel7">
```

Comment:
Change this to `container_all_services_disabled`

Reply:
Should this be all *system* services? May cause confusion when webservers etc run as a service inside the container.

Reply:
Initial idea for this rule was to have no services enabled at all, unless the container is specialized to do so. 

This specialized container could be single service, then start process of container would be the service itself. How to check that, PID 1 inside the container?

Or multi service container, in this case the start process would be systemd, but then, if you have systemd in container I don't see why we should prevent it from running all other services, there could be a variable to list services that should not be running.
This variable would probably carry system services and others we already marked as machine-only. It may be easier to have list of allowed services instead...